bench_util/Cargo.toml

```diff
@@ -4,11 +4,11 @@ name = "deno_bench_util"
 version = "0.7.0"
 authors = ["the Deno authors"]
 edition = "2018"
-description = "Bench and profiling utilities for deno crates"
 license = "MIT"
+publish = true
 readme = "README.md"
 repository = "https://github.com/denoland/deno"
-publish = true
+description = "Bench and profiling utilities for deno crates"
```

Comment:
`name` and `version` first, alphabetical, `description` last (https://github.com/rust-lang/rfcs/blob/master/style-guide/cargo.md#formatting-conventions)